### ISSUE TITLE
Cannot deploy panel with with HoloViews Tile to binder - cross origin policy

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/panel#2078]
Panel 0.11

I am still trying to get my Panel app deployed on binder :-) See 

- https://github.com/holoviz/panel/issues/2074
- https://github.com/holoviz/panel/issues/2066

Things are getting much better. But my app does still not work fully as the TILE is not displaying due to a missing "cross origin policy".

## Reproduce

https://mybinder.org/v2/gh/MarcSkovMadsen/load-forecasts/HEAD?urlpath=/panel/app

## Screenshot

![image](https://user-images.githubusercontent.com/42288570/110982980-7c96e280-8369-11eb-87aa-5f1b0a1f6f01.png)

FYI. @philippjfr

**Comment by philippjfr** [comment]
This has nothing to do with Panel, Bokeh or HoloViews. The tile provider simply has not allowed cross-origin requests, but has likely allowed localhost access which is why it works for you.

**Comment by MarcSkovMadsen** [comment]
Any info why this was closed?

**Comment by philippjfr** [comment]
This has nothing to do with Panel, Bokeh or HoloViews. The tile provider simply has not allowed cross-origin requests, but has likely allowed localhost access which is why it works for you.

**Comment by MarcSkovMadsen** [comment]
Is there any way to solve this issue?

**Comment by philippjfr** [comment]
No, none whatsoever. If this is one of the default tiles then we will have to remove it (we already know we have to do so for the Wikipedia tiles).

**Comment by philippjfr** [comment]
No, none whatsoever. If this is one of the default tiles then we will have to remove it (we already know we have to do so for the Wikipedia tiles).
